site/src/atomics/spacing.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Spacing Atomics
+description: Spacing related atomic css classes for the Atlas Design System
+template: standard
+---
+
+# Spacing Atomics
+
+Spacing Atomics can be used to add margin or padding on elements. They are especially helpful when composing layouts.
+
+## Class Pattern
+
+`<css-property-name>-<value>-<screensize>`
+
+For general information about the pattern, visit the [Atomics page](https://github.com/microsoft/atlas-design/blob/main/css/src/atomics/README.md).
+
+The following `<css-property-names>-<values>-<screensizes>` are supported.
+
+Accepted property names
+
+- `margin`, `padding`
+- `margin|padding-inline`
+- `margin|padding-block`
+- `margin|padding-top|right|bottom|left`
+
+Accepted values
+
+- `none, xs, s, m, l, xl, xxl, xxxl`
+
+Accepted screensizes
+
+- `tablet, desktop, widescreen`
+
+## Usages
+
+Adding uniform spacing
+
+Use `margin` with a size value to add spacing around an element.
+
+```html
+<div class="margin-xs border">XS</div>
```

Comment:
suggestion: rename `XS` to `margin-XS`? Same for the rest examples in this block

Reply:
That's what I had before, but Will suggested just using the sizes. I liked `margin-xs` better. @asrivastava0308 any thoughts?

Reply:
Yes, the code block will show the classes. We can reduce redundant information here by including the size.

Reply:
Actually, lets look at all the examples here in the sync today, and see if we can land on something good